Datasets/data-connect-dataset-outlookgroupconversations.md

```diff
@@ -41,29 +41,31 @@ The OutlookGroupConversations_v0 dataset can be joined with Mail, Message, Mail
 
 | Name  | Type  |  Description  |  FilterOptions  |  FilterType  |
 | ----------- | ----------- | ----------- | ----------- | ----------- |
-| Attachments |	string |	All attachments in a message. Contains metadata of attachments, such as LastModifiedDateTime, Name, ContentType, Size, IsInline, SourceUrl, and ProviderType. The actual attachment files are not included. |	No |	None |
+| Attachments |	array |	All attachments in a message. Contains metadata of attachments, such as LastModifiedDateTime, Name, ContentType, Size, IsInline, SourceUrl, and ProviderType. The actual attachment files are not included. |	No |	None |
 | BodyPreview	| string	| The first 255 characters of the message body content. |	No |	None |
-| CcRecipients |	string |	The Cc: recipients for the message. |	No |	None |
+| CcRecipients |	array |	The Cc: recipients for the message. |	No |	None |
 | ConversationId |	string |	The ID of the conversation that the email belongs to. |	No |	None |
 | ConversationIndex |	string |	Indicates the relative position of the message within the conversation that the message belongs to. |	No |	None |
 | CreatedDateTime |	datetime |	The date and time the message was created. |	Yes |	Date |
-| From |	string |	The mailbox owner and sender of the message. |	No |	None |
+| From |	object |	The mailbox owner and sender of the message. |	No |	None |
 | HasAttachments |	string |	Indicates whether the message has attachments. *Allowed values:* true or false. This property doesn't include inline attachments, so if a message contains only inline attachments, this property is false. |	No | 	None | 
 | Id |	string |	Unique identifier for the message. *Note:* This value may change if a message is moved or altered. |	No |	None |
 | Importance	| string |	The importance of the message; *Allowed values:* high, normal, low. |	No |	None |
 | InternetMessageId |	string |	The message ID in the format specified by RFC2822. |	No |	None |
 | IsRead |	string |	Indicates whether the message has been read. |	No |	None |
 | LastModifiedDateTime |	datetime |	The date and time the message was last changed. |	Yes |	Date |
-| MentionsPreview |	string |	Preview of the mentions in the message. |	No |	None |
-| ReplyTo |	string |	The email addresses to use when replying. |	No |	None |
-| Body |	string	| The UniqueBody in raw format. |	No |	None |
+| LikesPreview | object | Number of Likes of this message.| | 0 | false|
```

Comment:
Please remove "LikesPreview" entry, even though this is present in the json schema, we dont get it in the output, you can cross check that this field is not there in any sample data

Reply:
Removed